lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -736,6 +741,104 @@ class EnumSyntheticFrontEnd : public SyntheticChildrenFrontEnd {
   ConstString m_element_name;
   size_t m_child_index;
 };
+
+class TaskSyntheticFrontEnd : public SyntheticChildrenFrontEnd {
```

Comment:
Can you give this a doxygen comment explaining in a sentence or two what this is about?